tracing/tracing.go

```diff
@@ -146,3 +149,24 @@ func LogKV(k, v string, ctx context.Context) {
 		span.LogKV(k, v)
 	}
 }
+
+// GetTraceID retrieves TraceID from HTTP request, for example to search for this trace
+// in the UI of your tracing solution and to get more context about it
+func GetTraceID(req *http.Request) string {
+	span := ot.SpanFromContext(req.Context())
+	if span == nil {
+		return ""
+	}
+
+	spanContext := span.Context()
+	if spanContext == nil {
+		return ""
+	}
+
+	switch spanContextType := spanContext.(type) {
+	case origlightstep.SpanContext:
+		return fmt.Sprintf("0x%x", spanContextType.TraceID)
```

Comment:
Please add a comment why we format it as a hex with the `0x` prefix

Reply:
Decided to remove 0x prefix to span_id to be easy copy-pastable.